jujugui/static/gui/src/app/components/user-profile/entity/entity.js

```diff
@@ -254,7 +254,7 @@ YUI.add('user-profile-entity', function() {
       var isCharm = type === 'charm';
       // Model names will be in the format "username/model-name" so we have to
       // extract the part we need.
-      var name = isModel ? entity.name.split('/')[1] : entity.name;
```

Comment:
@hatched I have re-factored this to keep the split name by slash if the name contains a `/`. This raises the question of can model names contain `/` in there name?